README.md

```diff
@@ -8,16 +8,17 @@ This repository contains samples for usage of the Google Cloud Machine Learning
 
 ### CPU and GPU
 
-* [Keras Census](census/keras) - Keras single node model
-* [Census](census) - TensorFlow census models
-  * [Canned Estimator](census/estimator) - TensorFlow canned estimator model
-  * [Custom Estimator](census/customestimator) - TensorFlow custom estimator model
+* [Census](census) - Predict a person's income level
```

Comment:
Something like:
"Predict a person's income level" => "Predicting income class from Census data"

Reply:
It is more consistent with the Cloud Console docs as it stands.

Reply:
Also, as with the other items in the list, I have removed mention of the actual datasets at the top level in favor of a focus on the actual use case.